npm-inventory/src/index.ts

```diff
@@ -138,6 +138,8 @@ async function run (baseDir): Promise<void> {
             name: dep.name.split('/').pop(),
             license: dep.license,
             scope: dep.name.split('/')[0].replace(/^@/,''),
+            version: dep.version,
+            integrationSource: 'custom'
```

Comment:
what's the purpose of this?

Reply:
It's to distinguish the source of the entity in the graph, if users also have configured an NPM integration.

Reply:
But that can already be distinguished via `_source` property, right? This feels like a duplicate.

Reply:
Also, the the package already exists via the npm integration, will this map to it instead of creating a duplicate entity?

Reply:
Good point, re: `_source` property: it will say `api` instead of `integration-managed`.